Makefile

```diff
@@ -35,3 +41,89 @@ build-dependencies-axolotl:
 clean:
 	rm -f axolotl
 	rm -rf axolotl-web/dist
+
+check_platform_arm64:
```

Comment:
One last question. In the Makefile as it is now, each target is named with a dash where a space normally would be. Example `build-translation`. In your additions, you are using underscores. Example `check_platform_arm64`. Looking at the Makefile standard targets, they are all named with dashes as separator, for example `install-html`.

To keep the target naming the same across the file, could you also name the targets with a dash instead of underscore? It seems to me to be the standard Makefile target naming style.

https://www.gnu.org/software/make/manual/html_node/Standard-Targets.html

Reply:
So true, thanks for looking that up! Will change the target names.